src/Storage/ContentRequest/Save.php

```diff
@@ -348,7 +348,9 @@ private function createJsonUpdate(Entity\Content $content, $flush)
 
         $val = $content->toArray();
 
-        if (isset($val['datechanged'])) {
+        if ($val['datechanged'] instanceof Carbon) {
+            $val['datechanged'] = $val['datechanged']->toIso8601String();
+        } elseif (isset($val['datechanged'])) {
```

Comment:
Won't row 351 now raise `Notice: Undefined index` if `$val['datechanged']` is not set (`isset` returns false)?